airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestination.java

```diff
@@ -21,12 +22,13 @@ enum DestinationType {
     INTERNAL_STAGING
   }
 
-  public SnowflakeDestination() {
-    super(DestinationType.class, SnowflakeDestinationResolver::getTypeFromConfig, SnowflakeDestinationResolver.getTypeToDestination());
+  public SnowflakeDestination(final String airbyteEnvironment) {
+    super(DestinationType.class, SnowflakeDestinationResolver::getTypeFromConfig,
+        SnowflakeDestinationResolver.getTypeToDestination(airbyteEnvironment));
   }
 
   public static void main(final String[] args) throws Exception {
```

Comment:
probably best to just delete this method entirely, to avoid confusion?

Reply:
Yup, now it shouldn't be used anymore. Removed